addons/binding/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/handler/SomfyTahomaBridgeHandler.java

```diff
@@ -765,4 +755,7 @@ public String getTahomaStatus(String gatewayId) {
         return null;
     }
 
+    private boolean isAuthenticationChallenge(String message) {
```

Comment:
Would it make sense to let this message accept an exception, now every call contains repetition

Reply:
Done